packages/perseus/src/util/snowman-utils.ts

```diff
@@ -0,0 +1,149 @@
+export enum WidgetType {
+    Categorizer = "categorizer",
+    CSProgram = "cs-program",
+    Definition = "definition",
+    Dropdown = "dropdown",
+    Explanation = "explanation",
+    Expression = "expression",
+    GradedGroup = "graded-group",
+    GradedGroupAnswerBar = "graded-group-answer-bar",
+    GradedGroupSet = "graded-group-set",
+    Grapher = "grapher",
+    Group = "group",
+    Iframe = "iframe",
+    Image = "image",
+    InputNumber = "input-number",
+    Interaction = "interaction",
+    InteractiveGraph = "interactive-graph",
+    InteractiveNumberLine = "interactive-number-line",
+    LabelImage = "label-image",
+    Matcher = "matcher",
+    Matrix = "matrix",
+    Measurer = "measurer",
+    Molecule = "molecule",
+    NumberLine = "number-line",
+    NumericInput = "numeric-input",
+    Orderer = "orderer",
+    Passage = "passage",
+    PassageRef = "passage-ref",
+    PassageRefTarget = "passage-ref-target",
+    Plotter = "plotter",
+    Protractor = "protractor",
+    PythonProgram = "python-program",
+    Radio = "radio",
+    Sorter = "sorter",
+    Table = "table",
+    Unit = "unit",
+    Video = "video",
+    VideoTranscriptLink = "video-transcript-link",
+}
+
+const QUESTION_WIDGETS = [
```

Comment:
Is it possible to add a comment to explain what QUESTION WIDGETs are? I think it could help our future selves understand what these are. :) 

Iirc I THINK we determined that these are all widgets that actually require user input -- but not all of them.

Reply:
I'll add the comment "// These widgets all require user-input, though this is not an exhaustive list" or something like that. How does that sound?

Reply:
This was to help out with some of the cheat utils, correct? I wonder if something like: "These are all the currently widgets that require user-input and are supported for automatic scoring." would work? 

Hmm I wonder if we should document somewhere all these places where we're only supporting SOME widgets.🤔 It might be a good callout for potential places of improvement in the future.

Reply:
Could add a TODO above the places that require it and leave a comment about how only some widgets are listed and it will need to be updated in the future? 🤔